### PR TITLE
rename cloudId to hubId

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: charts/plgd-hub/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ global:
   # -- Global domain
   domain:
   # -- CloudID. Used by coap-gateway. It must be unique
```
Comment (Danielius1922) [inline charts/plgd-hub/README.md]: HubId?

File: http-gateway/swagger.yaml [hunk 2]
```diff
@@ -780,7 +780,7 @@ components:
         deviceOnboardingCodeUrl: 'https://auth.plgd.cloud/authorize?audience=http://try.plgd.cloud/'
         jwtClaimOwnerId: 'sub'
         signingServerAddress: 'api.try.plgd.cloud:443'
-        cloudId: 'adebc667-1f2b-41e3-bf5c-6d6eabc68cc6'
+        hubId: 'adebc667-1f2b-41e3-bf5c-6d6eabc68cc6'
         cloudUrl: 'coaps+tcp://try.plgd.cloud:5684'
```
Comment (Danielius1922) [inline http-gateway/swagger.yaml]: Wasn't `cloudUrl` changed to to `coapGateway` in the previous PR? Doesn't seem to be change in main either, must have missed this.

Comment (jkralik) [reply http-gateway/swagger.yaml]: Fixed

File: charts/plgd-hub/README.md (hunk 1 above)
Comment (jkralik) [reply charts/plgd-hub/README.md]: Fixed

**Review by Danielius1922 — APPROVED** [review]

File: charts/plgd-hub/values.yaml [hunk 3]
```diff
@@ -689,8 +689,8 @@ coapgateway:
     debug: false
     # -- Dump copp messages
     dumpCoapMessages: true
-  # -- Cloud ID. Can be override via global.cloudId
-  cloudId:
+  # -- Hub ID. Can be override via global.hubId
```
Comment (ondrejtomcik) [inline charts/plgd-hub/values.yaml:692]: @rafajpet ping, fetch new version

**Review by ondrejtomcik — APPROVED** [review]

**Pull request merged**